dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat_telemetry_metrics/test-applications/faulttolerancemetrics.jar/src/com/ibm/websphere/microprofile/faulttolerance/metrics/utils/CustomException.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
```

Comment:
2024

Reply:
I don't think we need to change this one, its copying a file unchanged